draft-ietf-core-groupcomm-bis.md

```diff
@@ -720,13 +724,15 @@ For observing at servers that are members of a CoAP group through a CoAP-to-CoAP
 
 {{Section 2.8 of RFC7959}} specifies how a client can use block-wise transfer (Block2 Option) in a multicast GET request to limit the size of the initial response of each server. Consistent with {{Section 2.5 of RFC8132}}, the same can be done with a multicast FETCH request.
 
-To retrieve any further blocks of the resource from responding servers, the client can rely on two possible approaches.
+If a client sends a multicast GET or FETCH request including a Block2 Option with a block number of 0, then the client can rely on two possible approaches in order to retrieve any further blocks of the resource from responding servers.
 
 1. The client uses unicast requests, separately addressing each different server.
 
-2. The client uses follow-up group requests, if all the responses received from different servers specify the same block size SIZE in their Block2 Option. In particular, SIZE can have the same value specified in the Block2 Option of the first group request, or instead a different one. If the client relies on this approach, follow-up group requests MUST specify SIZE in their Block2 Option.
+2. The client uses follow-up group requests, if all the responses received from different servers specify the same block size in their Block2 Option. In particular, such a block size can be equal to the block size specified in the Block2 Option of the first group request, or instead a smaller one. If the client relies on this approach, then the Block2 Option of follow-up group requests in the same block-wise transfer MUST specify the same block size used by all the servers in the Block2 Option of their responses.
```

Comment:
I'm still not sure the MUST is justified here (or is it intended as a restatement of something I'm missing?)

Reply:
It's a restatement of what is defined in Section 2.4 of of RFC 7959 (of course, there discussed for one-to-one communication). The same goes for the other "MUST" two paragraphs below.

https://github.com/core-wg/groupcomm-bis/pull/44/commits/d26b390a23cd2fe8816d872e32ef7ac3d4246e2a  addressed this by avoiding "MUST" and just asserting, i.e., s/MUST specify/specifies

Reply:
Right, that should be the latter :-)

Reply:
Marking this point as resolved, per the discussion and commit above